backstop/tests/checkbox.html

```diff
@@ -83,11 +83,11 @@ <h2>Default</h2>
               >
                 <path
                   class="iui-check"
-                  d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z"
+                  d="m6.5 12.5-4.5-4.5 1.5-1.5 3 3 6-6 1.5 1.5z"
                 />
                 <path
                   class="iui-check-partial"
-                  d="m1 6.5h14v3h-14z"
+                  d="m2.75 6.875h10.5v2.25h-10.5z"
```

Comment:
Why are all the svgs changed? I see no mention in the PR description

Reply:
The image below shows the new SVGs on the left & the old SVGs on the right.
![Screen Shot 2022-01-07 at 2 11 35 PM](https://user-images.githubusercontent.com/849817/148594838-9559d096-c843-4e50-a98e-d33933473e54.png)

We used to take those 16px SVGs and display them at `$iui-sm` (12px).  With the new method, we display the SVG at 100% (which happens to be 16px) so any of the empty space from the checkbox / radio button border need to be handled within the SVG itself.